opm/simulators/wells/BlackoilWellModel.hpp

```diff
@@ -215,14 +215,69 @@ namespace Opm {
                 return grp_nwrk_values;
             }
 
+            const WellStateFullyImplicitBlackoil& wellState() const
+            {
```

Comment:
Can you add some explaining text before the accessor functions.

Reply:
OK - I have tried to add some documentation. Hopefully it is clearer now.

Reply:
Thanks!